opm/output/eclipse/EclipseWriter.cpp

```diff
@@ -1205,6 +1223,13 @@ void EclipseWriter::writeInit(const SimulatorTimerInterface &timer)
             EclipseWriterDetails::restrictAndReorderToActiveCells(data, gridToEclipseIdx_.size(), gridToEclipseIdx_.data());
             fortio.writeKeyword("PERMZ", data);
         }
+        if (nnc.hasNNC()) {
+            std::vector<double> tran;
```

Comment:
Need to convert the `tran` vector - which is in SI units, to Eclipse output units. A better unit handling system is in the works from @jokva refactoring, but for now this should do:
```C++
#include <opm/parser/eclipse/ConversionFactors.hpp>

...

double conversionFactor;
if (eclipseState_->getDeckUnitSystem().getType == UnitSystem::UNIT_TYPE_METRIC)
    conversionFactor = 1.0 / Metric::Transmissibility;
else
   conversionFactor = 1.0 / Field::Transmissibility; 

....

tran.push_back( nd.trans * conversionFactor );
```